channeldb/kvdb/etcd/bucket.go

```diff
@@ -11,9 +11,9 @@ const (
 )
 
 var (
-	bucketPrefix   = []byte("b")
-	valuePrefix    = []byte("v")
-	sequencePrefix = []byte("$")
+	valuePostfix   = []byte{0x00}
+	bucketPostfix  = []byte{0xFF}
+	sequencePrefix = []byte("$seq$")
```

Comment:
Yes, sequence keys are omitted from `ForEach` because we fetch with the bucket id as the prefix, while for sequence keys the prefix is `$seq$` followed by the bucket id.